### ISSUE TITLE
Missing files

### ISSUE THREAD

**Issue by Simounet** [pixelfed/pixelfed#4441]
Hi @dansup ,
I might be wrong but it seems like there is some files missing. I can't run `npm run production` without any error because of these files missing:
```
resources/assets/sass/lib/manrope.scss (needed by resources/assets/js/admin.js)
resources/assets/components/admin/AdminDirectory.vue (needed by resources/assets/sass/landing.scss)
```

**Comment by mitexleo** [comment]
What are you trying to do ? 
You won't have to build anything to deploy Pixelfed.

**Comment by Simounet** [comment]
I'm trying to make improvements so I need to be able to build. 😁

**Comment by mitexleo** [comment]
> I'm trying to make improvements so I need to be able to build. 😁

Got it, sorry ! You should have mentioned that.

**Comment by duck2b** [comment]
Hi @neonota  

I am also encountering a similar problem now, why are files missing? Was it intentional? How do we need to solve it?


```
ERROR in ./resources/assets/js/admin.js 24:33-95
Module not found: Error: Can't resolve './../components/admin/AdminDirectory.vue' in '/Users/chunbo/www/pixelfed/resources/assets/js'
Did you miss the leading dot in 'resolve.extensions'? Did you mean '[".*",".wasm",".mjs",".js",".jsx",".json",".vue"]' instead of '["*",".wasm",".mjs",".js",".jsx",".json",".vue"]'?

ERROR in ./resources/assets/sass/landing.scss
Module build failed (from ./node_modules/_mini-css-extract-plugin@1.6.2@mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/_sass-loader@12.6.0@sass-loader/dist/cjs.js):
SassError: Can't find stylesheet to import.
  ╷
6 │ @import "lib/manrope";
  │         ^^^^^^^^^^^^^
  ╵
  resources/assets/sass/landing.scss 6:9  root stylesheet
    at processResult (/Users/chunbo/www/pixelfed/node_modules/_webpack@5.85.0@webpack/lib/NormalModule.js:764:19)
    at /Users/chunbo/www/pixelfed/node_modules/_webpack@5.85.0@webpack/lib/NormalModule.js:866:5
    at /Users/chunbo/www/pixelfed/node_modules/_loader-runner@4.3.0@loader-runner/lib/LoaderRunner.js:400:11
    at /Users/chunbo/www/pixelfed/node_modules/_loader-runner@4.3.0@loader-runner/lib/LoaderRunner.js:252:18
    at context.callback (/Users/chunbo/www/pixelfed/node_modules/_loader-runner@4.3.0@loader-runner/lib/LoaderRunner.js:124:13)
    at Object.loader (/Users/chunbo/www/pixelfed/node_modules/_sass-loader@12.6.0@sass-loader/dist/index.js:69:5)
    at runNextTicks (internal/process/task_queues.js:60:5)
    at processImmediate (internal/timers.js:437:9)

ERROR in /js/rempos
Module not found: Error: Can't resolve '/Users/chunbo/www/pixelfed/resources/assets/js/rempos.js' in '/Users/chunbo/www/pixelfed'
Did you miss the leading dot in 'resolve.extensions'? Did you mean '[".*",".wasm",".mjs",".js",".jsx",".json",".vue"]' instead of '["*",".wasm",".mjs",".js",".jsx",".json",".vue"]'?

ERROR in /js/rempro
Module not found: Error: Can't resolve '/Users/chunbo/www/pixelfed/resources/assets/js/rempro.js' in '/Users/chunbo/www/pixelfed'
Did you miss the leading dot in 'resolve.extensions'? Did you mean '[".*",".wasm",".mjs",".js",".jsx",".json",".vue"]' instead of '["*",".wasm",".mjs",".js",".jsx",".json",".vue"]'?

1 ERROR in child compilations (Use 'stats.children: true' resp. '--stats-children' for more details)
```

**Comment by mitexleo** [comment]
@dansup Kindly take a look at this..

**Comment by alexande-ru** [comment]
Same problem...

**Comment by alexande-ru** [comment]
Same problem...

**Comment by iamalibabaei** [comment]
same here

**Comment by dansup** [comment]
Fixed in #4465
